src/services/services.ts

```diff
@@ -4648,7 +4648,7 @@ module ts {
                     return true;
                 }
                 else if (parent.kind === SyntaxKind.BinaryExpression && (<BinaryExpression>parent).left === node) {
-                    var operator = (<BinaryExpression>parent).operator;
+                    var operator = (<BinaryExpression>parent).operatorToken.kind;

```

Comment:
Huh. For some reason I expected we'd use it more here; we should totally have getOccurrences support for binops